infrastructure/authentication/auth_service_impl.go

```diff
@@ -94,8 +125,13 @@ func (a *AuthenticationServiceImpl) UpdateCredentials(newToken string, sendNotif
 	c.SetToken(newToken)
 
 	if sendNotification {
-		a.notifier.Send(types.AuthenticationParams{Token: newToken})
+		a.c.Logger().Info().Str("method", "UpdateCredentials").Str(
+			"Sending Url %s",
+			a.c.SnykApi(),
+		)
```

Comment:
we need send a ShowMessage Notification here
example: Api Endpoint was updated from the server to: {apiUrl}